Breeze.UI/src/app/wallet/advanced/advanced.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="mainDiv">
+    <form [formGroup]='icoFormGroup'>
+        <div style="margin-top:20px">
```

Comment:
As per our discussion please tidy up all inline styles. There seems to be an abuse of those

Reply:
I have moved styles to classes where that style is used more than once.  If the style is used once then it's overkill.